.github/workflows/production-deploy.yml

```
@@ -0,0 +1,49 @@
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      # 1. Docker Hub에 로그인 (ECR 로그인 대신)
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)